### PR TITLE
renamed entry point to ZoldWts as in issue #8

### PR DESCRIPTION
**Pull request by ammaratef45**
PR for #8

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#12](https://github.com/amihaiemil/zold-java-client/issues/12) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#12](https://github.com/amihaiemil/zold-java-client/issues/12) is assigned to @denis-konovalyenko[/z](https://www.0crat.com/u/denis-konovalyenko), here is [why](https://www.0crat.com/footprint/GFCHY7NQG/5d89c7b7-69c4-402c-80ac-3ac856b331c4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by amihaiemil** [comment]
@ammaratef45 Please use the name ``ZoldWts`` (and ``RestfulZoldWts`` respectively), as I commented [here](https://github.com/amihaiemil/zold-java-client/issues/8#issuecomment-454311474). 

And please link the original issue in the PR's description. Just say "`PR for #8".

**Comment by denis-konovalyenko** [comment]
@ammaratef45 , as far as I understand, the expected change is `ZoldWts` instead of `Wts` was not it specified so in [this message](https://github.com/amihaiemil/zold-java-client/issues/8#issuecomment-454311474)?

@ammaratef45 , could you please check whether the commit message contains clear information on what kind of a problem has been fixed? I suppose you meant something like that "#8: the entry point renamed." and the "Wts ass" was just a typo, wasn't it?

@amihaiemil , what about QA rules, are we going to comply with the requirements stated in [the Policy](http://www.zerocracy.com/policy.html#42)? If so, then , @ammaratef45 , I would also mention the following point for considration: "Pull request description explains the solution proposed and contains a link to the original ticket it is related to".

**Comment by amihaiemil** [comment]
@denis-konovalyenko We don't have a QAs on the projects but yes, we should still try to respect the process as much as possible. However, we should not block PRs for small stuff like commit messages and so on.

**Comment by denis-konovalyenko** [comment]
@amihaiemil , thanks for the clarification! I have added a proposal to link this to the readme section (#13).

**Comment by ammaratef45** [comment]
@denis-konovalyenko sorry for the typo :laughing: 
I edited the name and the PR description.

File: .vscode/launch.test.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (denis-konovalyenko) [inline .vscode/launch.test.json]: @ammaratef45 , could you please clarify what these changes are for? Do we need them in the scope of the issue #8?

Comment (ammaratef45) [reply .vscode/launch.test.json]: @denis-konovalyenko this file should be removed
I was using vscode and it added this file automatically and I didn't notice it

Comment (ammaratef45) [reply .vscode/launch.test.json]: @denis-konovalyenko I have removed it in a seperate commit

**Comment by denis-konovalyenko** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @denis-konovalyenko[/z](https://www.0crat.com/u/denis-konovalyenko)

**Comment by 0crat** [comment]
Job `gh:amihaiemil/zold-java-client#12` doesn't exist in WBS, can't create order

**Comment by 0crat** [comment]
Job `gh:amihaiemil/zold-java-client#12` doesn't exist in WBS, can't create order

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**